### DISCUSSION TITLE
Designing gap-fill medium for rhizosphere microbes

### DISCUSSION THREAD

**Discussion by arijitnus** [jotech/gapseq#194] [Q&A]
Dear researchers, 

I am interested in designing a gapfiling media for rhizosphere microbes grown in plant associations. The experimental condition is such that bacteria is able to access nutrients from root exudates (of Arabidopsis plants) and MS media (Murashine-skoog) for plant growth. Whereas I know the composition of the media + root exudates, I do not have experimental data for max flux rate. Could anyone guide on how could we generate such custom gap filling media ? Any microbiological media closer to the media mentioned above could also be a lead. 

Appreciate any guidance on this

**Comment by jotech** [comment]
hi @arijitnus 
for the gap-filling, the exact flux values are not necessary. Our predefined media  [1] usually define non-limiting substances as 100 and balance carbon and nitrogen sources (and eventually electron acceptors) to reasonable physiological (much lower) proportions.
For flux balance analysis in general, however, the medium definition has a huge effect.
Some additional information can be found here [2], and a related publication is also available [3]

[1] https://github.com/jotech/gapseq/tree/master/dat/media
[2]  https://gapseq.readthedocs.io/en/latest/usage/medium.html
[3] https://doi.org/10.1371/journal.pone.0236890

**Comment by arijitnus** [comment]
Thanks @jotech . These references were helpful.
